### PR TITLE
Mention SELinux in README Docker section

### PR DESCRIPTION
**Pull request by dekimir**
## Description

Update README to suggest turning SELinux off, to get rid of errors like this:
```
Configuration error:
config directory doesn't contain a conf.py file (/docs/src/main/sphinx)
```

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ information about reporting vulnerabilities.
 * Mac OS X or Linux
 * Java 17.0.4+, 64-bit
 * Docker
+  * Turn SELinux off on the build host, allowing containers to mount parts of the Trino source tree
```
Comment (mosabua) [inline README.md:39]:
```suggestion
  * Turn SELinux or other systems disabling write access to the local checkout off, to allow containers to mount parts of the Trino source tree
```

**Review by mosabua — CHANGES_REQUESTED** [review]

**Comment by mosabua** [comment]
Will also need a signed CLA @dekimir

**Comment by dekimir** [comment]
@mosabua I discovered that I can add a `docker` flag to exempt the build container from security policies -- please see [here](https://github.com/dekimir/trino/commit/d2aed30a24bd6340d7de63c4a6a39dfe3f78f784).  It might be preferable to commit that instead of this PR (though it'll need more testing; I only tested it on my Fedora 38 box).

Also, I emailed the signed CLA to the email address mentioned by the bot above.

**Comment by mosabua** [comment]
That could docker script mod could be another PR .. but given that there are other reasons for disabled write access this PR is still valid

File: README.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ information about reporting vulnerabilities.
 * Mac OS X or Linux
 * Java 17.0.4+, 64-bit
 * Docker
+  * Turn SELinux or other systems disabling write access to the local checkout off, to allow containers to mount parts of the Trino source tree
```
Comment (mosabua) [inline README.md:39]: can you wrap at 80 char column please

Comment (dekimir) [reply README.md]: Done in dce1fa8.

**Comment by dekimir** [comment]
> That could docker script mod could be another PR

#19940

**Comment by martint** [comment]
@cla-bot check

**Review by mosabua — APPROVED** [review]
Looks good now. Thank you.

**Pull request merged**